src/components/team.js

```diff
@@ -1,11 +1,77 @@
 import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
 import "./team.scss"
 import { useWindowSize } from "./hooks"
 
 import TeamMember from "./teamMember"
 
 export default function Team({ team }) {
   const { width } = useWindowSize()
+
+  const data = useStaticQuery(graphql`
+    {
+      allContentfulListOfThings(
+        filter: { listName: { eq: "About page bios" } }
+      ) {
+        totalCount
+        nodes {
+          entries {
+            ... on ContentfulBio {
+              role
+              name
+              description {
+                content {
+                  content {
+                    value
+                    marks {
+                      type
+                    }
+                  }
+                }
+              }
+              image {
+                fixed {
+                  src
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const teamMembers = data.allContentfulListOfThings.nodes[0].entries.reduce(
```

Comment:
It looks like this the pretty standard way you are pulling and organizing data from Contentful. I'm thinking it might be good to just add some comments here to help describe what is going on and why you are doing it. Just jumping in and looking at this code it is pretty difficult to parse.

Reply:
Yeah, I agree, I'll add some comments in there. That is a tricky one to follow.

Reply:
I agree that's a tricky one to follow. I'll add some comments to that one.